presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestIcebergDistributed.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.iceberg;
+
+import com.facebook.presto.testing.MaterializedResult;
+import com.facebook.presto.tests.AbstractTestDistributedQueries;
+import com.google.common.collect.ImmutableMap;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.common.type.VarcharType.VARCHAR;
+import static com.facebook.presto.testing.MaterializedResult.resultBuilder;
+import static com.facebook.presto.testing.assertions.Assert.assertEquals;
+
+@Test
+public class TestIcebergDistributed
+        extends AbstractTestDistributedQueries
+{
+    public TestIcebergDistributed()
+    {
+        super(() -> IcebergQueryRunner.createIcebergQueryRunner(ImmutableMap.of()));
+    }
+
+    @Override
+    protected boolean supportsViews()
+    {
+        return false;
+    }
+
+    @Override
+    protected boolean supportsNotNullColumns()
+    {
+        return false;
+    }
+
+    public void testJoinWithLessThanOnDatesInJoinClause()
+    {
+    }
+
+    @Override
+    public void testRenameTable()
+    {
```

Comment:
leave a comment if the rename table feature has not been supported yet,  or add a todo if it's already supported

Reply:
Yeah, we don't support it yet. The reason we have the function is that we need to override the function defined in the `AbstractTestDistributedQueries`. We leverage that class to test SQL queries against the connector.